examples/fireworks_server/build_database.py

```diff
@@ -3,7 +3,7 @@
 
 # Builds a blank database object
 # Tell the database we are going to build interaction energies
-db = portal.collections.Database("Water", db_type="ie")
+db = portal.collections.Dataset("Water", db_type="ie")
```

Comment:
Can you change this variable to be `ds` or something? Just so its more clear this is the Dataset and there is zero ambiguity between the Mongo Database?